pkg/ovs/daemonset.go

```diff
@@ -78,16 +79,15 @@ func DaemonSet(
 	envVars["OvnBridge"] = env.SetValue(instance.Spec.ExternalIDS.OvnBridge)
 	envVars["OvnRemote"] = env.SetValue(dbmap["SB"])
 	envVars["OvnEncapType"] = env.SetValue(instance.Spec.ExternalIDS.OvnEncapType)
+	// TODO
+	envVars["PodNamespace"] = env.SetValue(instance.Namespace)
+	envVars["PodNetworksStatus"] = EnvDownwardAPI("metadata.annotations['k8s.v1.cni.cncf.io/networks-status']")
+	envVars["OvnEncapNetwork"] = env.SetValue("internalapi")
```

Comment:
this needs update to not be fixed for sure and get it from the NetworkAttachment